recipes/mumps/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ `uname` == "Darwin" ]; then
+  cp $RECIPE_DIR/Makefile.debian.SEQ_mac Makefile.inc
+  export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+  cp $RECIPE_DIR/Makefile.debian.SEQ Makefile.inc
```

Comment:
By searching for `./ssimpletest` in the [circle-CI build report](https://circleci.com/gh/conda-forge/staged-recipes/10409?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) you see that it also worked without adding that line.